docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -2138,13 +2142,58 @@ public class CustomGlobalFilter implements GlobalFilter, Ordered {
 ----
 ====
 
+=== The Gateway Metrics Filter
+
+To enable gateway metrics, add spring-boot-starter-actuator as a project dependency. Then, by default, the gateway metrics filter runs as long as the property `spring.cloud.gateway.metrics.enabled` is not set to `false`. This filter adds a timer metric named `spring.cloud.gateway.requests` with the following tags:
+
+* `routeId`: The route ID.
+* `routeUri`: The URI to which the API is routed.
+* `outcome`: The outcome, as classified by link:https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpStatus.Series.html[HttpStatus.Series].
+* `status`: The HTTP status of the request returned to the client.
+* `httpStatusCode`: The HTTP Status of the request returned to the client.
+* `httpMethod`: The HTTP method used for the request.
+
+In addition, through the property `spring.cloud.gateway.metrics.tags.path.enabled` (by default, set to false), you can activate an extra metric with the tag:
```

Comment:
Change to:

In addition, through the `spring.cloud.gateway.metrics.tags.path.enabled` property (by default, `false`), you can activate an extra metric with the `path` tag: